### ISSUE TITLE
Schema is missing payload contents

### ISSUE THREAD

**Issue by btorresgil** [octokit/webhooks#43]
In the generated schema there is only `name`, `actions`, and `examples`. The **payload** of the webhook is not pulled from the documentation.  This results in major consequences down the chain as [webhooks.js](https://github.com/octokit/webhooks.js) is derived from this, as is all the [TypeScript typings](https://github.com/octokit/webhooks.js/blob/master/index.d.ts) for the webhooks.

This means that critical payload values that exist in the webhook are not accessible in TypeScript without some terrible escape hatches.

For example, in an IssuesEvent, a developer trying to access the `label` payload property would not get prompted that it exists:

<img width="556" alt="2019-09-25_13-24-32" src="https://user-images.githubusercontent.com/4164289/65636931-144b1a00-df98-11e9-97cb-6b492441c22f.png">

And entering it anyway results in a compiler error, even though the property exists in the webhook:

<img width="692" alt="2019-09-25_13-25-00" src="https://user-images.githubusercontent.com/4164289/65636982-3775c980-df98-11e9-9dd8-5b1468f00292.png">

The only workaround I've found is to override the types to add the missing properties when they are needed, or escape hatch with `any`:

```ts
webhooks.on('issues.labeled', ({payload}: {payload: any}) => {
  const label = payload.label
})
```

Since using `any` removes any type checking, it kinda defeats the purpose of having a schema. So it would be great if the schema could be complete enough to include all the payload parameters in the documentation.

Thanks!

**Comment by btorresgil** [comment]
Also created issue octokit/webhooks.js#94 which depends on this issue.

**Comment by gr2m** [comment]
The examples are payload examples and the Typescript definitions are later derived from these. Ideally we would have JSON Schema specifications for each webhook event but that's not happening anytime soon.

They payloads look different for different cases, sometime keys are included and sometimes not. The best we can do is to kindly ask support@github.com to extend the respective payload example.

What I would love to do is setup an automated recording of webhooks similar to how we record fixtures at https://github.com/octokit/fixtures, that way we can just add more scenarios to record more example payloads. But that will require a pretty major effort.

We are working on an OpenAPI spec for the REST APIs right now, once we have that, I hope we can work on JSON Specs for the webhooks next.

**Comment by gr2m** [comment]
Another thing we could do is to derive a JSON Schema from the payload tables as you suggest and publish these next to the examples. I'd happily accept a pull request

**Comment by wolfy1339** [comment]
This should be resolved by #186

**Comment by gr2m** [comment]
I think so, too. Please let us know if the problem persists
